Tests/ToolBox/ForexVolume/ForexVolumeDownloaderTest.cs

```diff
@@ -0,0 +1,226 @@
+﻿using Ionic.Zip;
+using NodaTime;
+using NUnit.Framework;
+using QuantConnect.Data;
+using QuantConnect.Data.Custom;
+using QuantConnect.ToolBox;
+using QuantConnect.ToolBox.FxVolumeDownloader;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace QuantConnect.Tests.ToolBox.FxVolume
```

Comment:
In that case I would remove the `FxVolume` from the namespace entirely and just use `QuantConnect.Tests.ToolBox` as the namespace